docs/css/custom.css

```diff
@@ -2,6 +2,10 @@ body {
     background-color: #141B2B;
 }
 
+.fix-shadow {
+    box-shadow: none;
+}

```

Comment:
It looks better without the shadow. 

You can remove it at  https://github.com/zodern/meteor-up/blob/f11970f9967b215b53216a81b3ffa7f2f0da7f1c/docs/css/custom.css#L156-L161

Reply:
Oh, I totally missed that.